app/src/main/res/values-fr/strings.xml

```diff
@@ -115,12 +132,21 @@
     <string name="suffix_seekbar_deadzone">%</string>
     <string name="title_checkbox_xb1_driver">Pilote de contrôleur Xbox 360/One</string>
     <string name="summary_checkbox_xb1_driver">Active un pilote USB intégré pour les périphériques sans prise en charge du contrôleur Xbox natif.</string>
+    <string name="title_checkbox_usb_bind_all">Ignorer le support du contrôleur Android</string>
+    <string name="summary_checkbox_usb_bind_all">Force le pilote USB de Moonlight à prendre en charge tous les gamepads Xbox pris en charge</string>
+    <string name="title_checkbox_mouse_emulation">Emulation de la souris via le gamepad</string>
+    <string name="summary_checkbox_mouse_emulation">Appuyez longuement sur le bouton Start pour faire basculer la manette de jeu en mode souris.</string>
 
     <string name="category_on_screen_controls_settings">Paramètres des contrôles à l\'écran</string>
     <string name="title_checkbox_show_onscreen_controls">Afficher les commandes à l\'écran</string>
     <string name="summary_checkbox_show_onscreen_controls">Afficher la superposition du contrôleur virtuel sur l\'écran tactile</string>
     <string name="title_only_l3r3">Montre seulement L3 et R3</string>
     <string name="summary_only_l3r3">Cacher tout sauf L3 et R3</string>
+    <string name="title_reset_osc">Effacer la disposition des commandes à l\'écran sauvegardée</string>
```

Comment:
I just noticed that the tense of the verbs is not the same throughout the file. 
(ignor**er** vs. ignor**ez**, etc.). 
Mon français n'est plus très bon... Ça fait longtemps ;) Avez vous un opinion?

Reply:
Pour effacer il faut dire mordre pour er et mordu pour é enfin moi j'ai appris comme ça
et donc ici c'est " mordre la disposition des commandes....." = Effacer 
Sinon pour Effacez c'est pour "vous" = Vous effacez
Excuse moi je le dit en français